governance/index.html

```diff
@@ -0,0 +1,315 @@
+
+                  <p>“Core components” are those that will likely be used by a very high proportion of the community, “extensions” are where more than one community member could reuse them and “compatible components” are those that are more specific to a single organisation.</p>
+
+                  <p>The content creator and eLife will decide how a component should be classified and components are likely to move from compatible, through extension to core as reuse increases and the platform evolves. The Product SIG will advise on classification and movement between classes for components.</p>
+
+                  <p>Examples of each class might include:</p>
+                   <ul>
+                     <li>Core - content store, schema for article title or abstract, patterns for article title or abstract</li>
+                     <li>Extension - IIIF image server, schema for plain language summary, about pages</li>
+                     <li>Compatible - adaptor to send content to a publisher’s proprietary systems</li>
+                   </ul>
+
+
+                </div>
+
+              </section>
+
+              <section class="article-section" data-behaviour="ArticleSection">
+
+                <header class="article-section__header">
+                  <h2 class="article-section__header_text">Trademark Licencing</h2>
+                </header>
+
+                <div class="article-section__body">
+                  <p>eLife is pursuing a trademark for the use of Libero with respect to downloadable software and service provision for the general functions of the software.</p>
+
+                  <p>The protection this affords will allow the community to have more control over the use of the name in relation to publishing to ensure the use aligns with the community’s values. It also helps prevent bad actors from promoting an association with the community and its members.</p>
+
+                  <p>While eLife encourages community members to promote their use of Libero with no restrictions, they also offer the option to licence the trademark to organisations that wish to leverage the name with their usage. This might be of interest to organisations that wish to build their  reputation in the publishing industry or to provide services using Libero, for example. A process for applying to use the trademark will be decided and made available here when it is ready.</p>
+
+                </div>
+
+              </section>
+
+              <section class="article-section" data-behaviour="ArticleSection">
+
+                <header class="article-section__header">
+                  <h2 class="article-section__header_text">Dispute Resolution</h2>
+                </header>
+
+                <div class="article-section__body">
+                  <p>As suggested by <a href="http://oss-watch.ac.uk/resources/benevolentdictatorgovernancemodel">OSS Watch</a>:</p>
+
+                  <blockquote class="pull-quote">
```

Comment:
Less fancy, but AFAIK https://ui-patterns.elifesciences.org/?p=molecules-quote is more meant for including content from somewhere else. (As opposed to something someone said.)

Reply:
Changed to class="quote"